main.star

```diff
@@ -1,22 +1,49 @@
 validator = import_module("github.com/kurtosis-tech/hyperlane-package/validator.star")
 relayer = import_module("github.com/kurtosis-tech/hyperlane-package/relayer.star")
 
-def run(plan,
-    origin_chain,
+def run(
+    plan,
+    origin_chain_name, #type: string
+    origin_chain_url,  #type: string
+    validator_key,     #type: string
     remote_chains,
-    aws_env={}
+    aws_access_key_id="",  #type: string
+    aws_secret_access_key="", #type: string
+    aws_bucket_region="", #type: string
+    aws_bucket_name="", #type: string
+    aws_bucket_folder="" #type: string
```

Comment:
This is very suboptimal IMO. Can't we keep it as its own object as it is optional? It's very ugly like this

Reply:
@gbouv  the reason for me to do this because in the ui, i would like to be as verbose as possible -- that is users know what fields to enter.. with json objects the users wouldnt know the knowledge of json schema